pytest_django/fixtures.py

```diff
@@ -192,6 +195,63 @@ def transactional_db(request, _django_db_setup, _django_cursor_wrapper):
     return _django_db_fixture_helper(True, request, _django_cursor_wrapper)
 
 
+@pytest.fixture(scope='session')
+def shared_db_wrapper(_django_db_setup, _django_cursor_wrapper):
+    """Wrapper for common database initialization code.
+
+    This fixture provides a context manager that let's you access the database
+    from a transaction spanning multiple tests.
+    """
+    from django.db import connection, transaction
+
+    if get_django_version() < (1, 6):
+        raise Exception('shared_db_wrapper is only supported on Django >= 1.6.')
+
+    class DummyException(Exception):
+        """Dummy for use with Atomic.__exit__."""
+
+    @contextmanager
+    def wrapper(request):
+        # We need to take the request
+        # to bind finalization to the place where this is used
+        if 'transactional_db' in request.funcargnames:
+            raise Exception(
+                'shared_db_wrapper cannot be used with `transactional_db`.')
+
+        with _django_cursor_wrapper:
+            if not connection.features.supports_transactions:
+                raise Exception(
+                    "shared_db_wrapper cannot be used when "
+                    "the database doesn't support transactions.")
+
```

Comment:
It seems to me that the next 25 lines could we written as:

        try:
            _django_cursor_wrapper.enable()
            with transaction.atomic():
                yield
                transaction.set_rollback(True)
        finally:
            _django_cursor_wrapper.restore()

This raises the question of why this fixture needs to care about `_django_cursor_wrapper`; it's unclear to me why this is necessary.